tests/metrics/collect_mem_consmd.sh

```diff
@@ -274,24 +366,32 @@ function main()
 
 	echo "Active gate"
 	echo "True" > $HOST_GATE
+	sleep 0.5
+	start_psstop
 
 	while [ 1 ];
 	do
 		cont_num=$(docker ps | wc -l)
 		if (( $cont_num == 1 ));then
+			stop_psstop
 			break
 		fi
 
 		sleep 0.1
 	done
 
 	echo "Workload finishes"
+	sleep 1
```

Comment:
It was for taking some samples after workload finishes, but at this moment it is unnecessary